docs/installation.rst

```diff
@@ -0,0 +1,424 @@
+-  ``BUILD_BENCHMARK`` - include building benchmarks (default: ON)
+-  ``BUILD_LMDB`` - build with support for LMDB (default: OFF)
+-  ``BUILD_NVTX`` - build with NVTX profiling enabled (default: OFF)
+-  ``BUILD_TENSORFLOW`` - build TensorFlow plugin (default: OFF)
+-  ``BUILD_NVJPEG`` - build with ``nvJPEG`` support (default: ON)
+-  ``BUILD_NVOF`` - build with ``NVIDIA OPTICAL FLOW SDK`` support (default: ON)
+-  ``BUILD_NVDEC`` - build with ``NVIDIA NVDEC`` support (default: ON)
+-  ``BUILD_NVML`` - build with ``NVIDIA Management Library`` (``NVML``) support (default: ON)
+-  ``WERROR`` - treat all build warnings as errors (default: OFF)
+-  ``BUILD_WITH_ASAN`` - build with ASAN support (default: OFF). To run issue: LD_LIBRARY_PATH=.
+   ASAN_OPTIONS=symbolize=1:protect_shadow_gap=0 ASAN_SYMBOLIZER_PATH=$(shell which llvm-symbolizer)
+   LD_PRELOAD=*PATH_TO_LIB_ASAN*/libasan.so.*X* *PATH_TO_BINARY*
+   where *X* depends on used compiler version (for example GCC 7.x uses 4). Tested with GCC 7.4, CUDA 10.0
+   and libasan.4. Any earierl version may not work.
+-  ``DALI_BUILD_FLAVOR`` - Allow to specify custom name sufix (i.e. 'nightly') for nvidia-dali whl package
+-  *(Unofficial)* ``BUILD_JPEG_TURBO`` - build with ``libjpeg-turbo`` (default: ON)
+
+.. note::
+
+   DALI release packages are built with the options listed above set to ON and NVTX turned OFF.
+   Testing is done with the same configuration.
+   We ensure that DALI compiles with all of those options turned OFF, but there may exist
+   cross-dependencies between some of those features.
+
+Following CMake parameters could be helpful in setting the right paths:
+
+.. |libjpeg-turbo_cmake link| replace:: **libjpeg CMake docs page**
+.. _libjpeg-turbo_cmake link: https://cmake.org/cmake/help/v3.11/module/FindJPEG.html
+.. |protobuf_cmake link| replace:: **protobuf CMake docs page**
+.. _protobuf_cmake link: https://cmake.org/cmake/help/v3.11/module/FindProtobuf.html
+
+* FFMPEG_ROOT_DIR - path to installed FFmpeg
+* NVJPEG_ROOT_DIR - where nvJPEG can be found (from CUDA 10.0 it is shipped with the CUDA toolkit so this option is not needed there)
+* libjpeg-turbo options can be obtained from |libjpeg-turbo_cmake link|_
+* protobuf options can be obtained from |protobuf_cmake link|_
+
+Install Python bindings
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    pip install dali/python
+
+Cross-compiling DALI C++ API for aarch64 Linux (Docker)
+-------------------------------------------------------
+Build the aarch64 Linux Build Container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    docker build -t dali_builder:aarch64-linux -f Dockerfile.build.aarch64-linux .
+
+Compile
+^^^^^^^
+From the root of the DALI source tree
+
+.. code-block:: bash
+
+    docker run -v $(pwd):/dali dali_builder:aarch64-linux
+
+The relevant artifacts will be in ``build/install`` and ``build/dali/python/nvidia/dali``
+
+Cross-compiling DALI C++ API for aarch64 QNX (Docker)
```

Comment:
We do not have QNX yet, this part was removed in some other PR because it got to docs by mistake.